dotcom-rendering/src/web/components/Dropdown.tsx

```diff
@@ -77,7 +77,6 @@ const ulStyles = css`
 
 const liStyles = css`
 	position: relative;
-	display: flex;
```

Comment:
This was causing the dropdown items to 'shrink' which meant that their click areas no longer took up the full box: 
<img width="197" alt="image" src="https://user-images.githubusercontent.com/9575458/235911434-eb3f86c8-0910-4ccf-b1e5-db5e2338e41e.png">